1619657740715 [Opening serial port: /dev/ttyTHS1.] 
1619657741017 [The robot is connected to ] [/dev/ttyTHS1] 
1619657741017 [Initial robot's topic...] 
1619657754347 [Data from: [w]] 
1619657755856 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657755856 [Data from: [ ]] 
1619657760366 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657760366 [Data from: [w]] 
1619657761872 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657761872 [Data from: [ ]] 
1619657766402 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657766402 [Data from: [w]] 
1619657767905 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657767905 [Data from: [ ]] 
1619657772419 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657772419 [Data from: [w]] 
1619657773926 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657773926 [Data from: [ ]] 
1619657778445 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657778445 [Data from: [w]] 
1619657779950 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657779950 [Data from: [ ]] 
1619657784457 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657784457 [Data from: [w]] 
1619657785959 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657785959 [Data from: [ ]] 
1619657790466 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657790466 [Data from: [w]] 
1619657791969 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657791969 [Data from: [ ]] 
1619657796476 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657796476 [Data from: [w]] 
1619657797978 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657797978 [Data from: [ ]] 
1619657802485 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657802485 [Data from: [w]] 
1619657803987 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657803987 [Data from: [ ]] 
1619657808505 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657808505 [Data from: [w]] 
1619657810017 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657810017 [Data from: [ ]] 
1619657814368 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 79, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814369 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657814369 [Data from: [w]] 
1619657814369 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814369 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814369 [Data from: [ ]] 
1619657814371 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814371 [Data from: [w]] 
1619657814371 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814371 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814371 [Data from: [ ]] 
1619657814372 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814372 [Data from: [w]] 
1619657814372 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814372 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814372 [Data from: [ ]] 
1619657814373 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814373 [Data from: [w]] 
1619657814373 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814373 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814373 [Data from: [ ]] 
1619657814373 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814373 [Data from: [d]] 
1619657814374 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 92, in handle_command
    self.set_speed([self._current_speed,self._current_speed*0.5])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814374 [Data from: [ ]] 
1619657814374 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814375 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657814375 [Data from: [w]] 
1619657814375 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814375 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814375 [Data from: [ ]] 
1619657814377 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814377 [Data from: [w]] 
1619657814378 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814379 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814379 [Data from: [ ]] 
1619657814380 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814380 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657814380 [Data from: [w]] 
1619657814381 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814381 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814381 [Data from: [ ]] 
1619657814382 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814383 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657814383 [Data from: [w]] 
1619657814383 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814383 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814383 [Data from: [ ]] 
1619657814389 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814389 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 5 \ x 0 2 N \ n \ r ' ) ]] 
1619657814389 [Data from: [w]] 
1619657814390 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 102, in handle_command
    self.set_speed([self._current_speed, self._current_speed])
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 300, in set_speed
    self.__robot_serial.set_speed(speed_data)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 104, in set_speed
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
1619657814390 [log_latest_command] [[ b y t e a r r a y ( b ' \ x a a \ x f f \ x a 2 \ x a 0 \ x 0 f \ x 0 0 \ x a 0 \ x 0 f \ x 0 0 \ x 0 3 \ x a 9 \ n \ r ' ) ]] 
1619657814390 [Data from: [ ]] 
1619657814390 [handle_key_command@Exception] [Traceback (most recent call last):
  File "/home/khoixx/dev_ros1_ws/src/execute/src/Robot.py", line 96, in handle_command
    self.release_motor()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_control.py", line 206, in release_motor
    self.__robot_serial.set_stop()
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 109, in set_stop
    self.write_command(command)
  File "/home/khoixx/dev_ros1_ws/src/execute/src/robot_command.py", line 73, in write_command
    self.__robot_serial.write(command_send)
  File "/usr/local/lib/python2.7/dist-packages/serial/serialposix.py", line 655, in write
    raise SerialException('write failed: {}'.format(e))
SerialException: write failed: [Errno 5] Input/output error
] 
